src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -164,7 +164,7 @@ export function registerTerminalActions() {
 			const commandService = accessor.get(ICommandService);
 
 			let event: MouseEvent | PointerEvent | KeyboardEvent | undefined;
-			let options: ICreateTerminalOptions | undefined;
```

Comment:
Think we still need this `| undefined`, otherwise the `if (options) { ... } else { ... }` below won't hit the else case